### PR TITLE
Upgrade eslint and add .vue file support

### PR DESCRIPTION
**Pull request by marcbachmann**

### CODE REVIEW

**Comment by benib** [comment]
@marcbachmann could you add https://www.npmjs.com/package/eslint-plugin-vue as well?
We can then extend the rules defined in there...

**Comment by marcbachmann** [comment]
thanks. had that first, but executed it in the wrong directory.

**Comment by benib** [comment]
👍

**Comment by benib** [comment]
@marcbachmann can you merge this? I guess the builds here are failing because of missing vue eslint support: https://drone.livingdocs.io/livingdocsIO/livingdocs-editor/7310/1/2

File: eslint/package.json [hunk 1]
```diff
@@ -10,18 +10,23 @@
...
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jest": "^23.8.1",
+    "eslint-plugin-jsx": "^0.1.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-vue": "^6.2.1",
+    "vue-eslint-parser": "^7.0.0"
```
Comment (benib) [inline eslint/package.json:30]: afaik this is installed as a dependency from `eslint-plugin-vue`. So there is probably no need to have it here.

**Pull request merged**